CMakeLists.txt

```diff
@@ -88,12 +88,12 @@ include(policies)
 #################################################################################
 find_program(EXE_SQLITE3 sqlite3)
 if (NOT EXE_SQLITE3)
-MESSAGE (SEND_ERROR "sqlite3 binary not found!")
+MESSAGE (FATAL_ERROR "sqlite3 binary not found!")
 ENDIF (NOT EXE_SQLITE3)
 
 FIND_PACKAGE(Sqlite3)
 IF (NOT SQLITE3_FOUND)
-MESSAGE (SEND_ERROR "sqlite3 dependency not found!")
+MESSAGE (FATAL_ERROR "sqlite3 dependency not found!")
 ENDIF (NOT SQLITE3_FOUND)
```

Comment:
Unlike `find_program`, `find_package` supports a `REQUIRED` argument which stops processing with an error message if the package cannot be found. (And this is implemented in `cmake/FindSqlite3.cmake`.)

Reply:
I tested this, and it does not stop a is documented/expected. CMake shows the message and continues. [Some other folks appear to see this behaviour too](https://stackoverflow.com/a/2711683), with the implication that `FindSqlite3.cmake` is potentially broken. This modification to message seems to work fine.

Reply:
I don't understand.

> I tested this, and it does not stop as is documented/expected. CMake shows the message and continues.

This is expected. CMake has to continue in order to "collect more errors, before failing.". But eventually it will end with an error. This allows to report both the missing executable and package, instead of stopping already at the missing executable.

> Some other folks appear to see this behaviour too, with the implication that FindSqlite3.cmake is potentially broken.

I can't see how this is related. The linked Q&A is from 2010 and neither adressing this project nor SQLite. For concerns about `REQUIRED` not being support cf.:
https://github.com/OSGeo/proj.4/blob/a813ad65cfb819bfa295dfbacdf577b258377c93/cmake/FindSqlite3.cmake#L60-L66

Reply:
I'll revert the message level back to `SEND_ERROR`, and add `REQUIRED` as recommended. I initially found it confusing that CMake would fail to find a required dependency, then continue to download googletest and perform other checks as though everything were normal.

Reply:
IMHO, I would suggest avoid last-minute pre-release fiddling with `FindSQLite3.cmake`, unless you are deadly seriously certain you know what you are doing.

Reply:
> IMHO, I would suggest avoid last-minute pre-release fiddling with `FindSQLite3.cmake`, unless you are deadly seriously certain you know what you are doing.

Agreed. The only modification is to add `REQUIRED` to `FIND_PACKAGE`, which seems to have been since old versions.